DB/src/main/java/io/deephaven/db/tables/lang/DBLanguageParser.java

```diff
@@ -356,15 +356,16 @@ private Method getMethod(final Class scope, final String methodName, final Class
         final ArrayList<Method> acceptableMethods = new ArrayList<>();
 
         if (scope==null){
-            for (final Class classImport : staticImports){
-                for (Method method : classImport.getDeclaredMethods()){
+            for (final Class classImport : staticImports) {
+                for (Method method : classImport.getDeclaredMethods()) {
                     possiblyAddExecutable(acceptableMethods, method, methodName, paramTypes, parameterizedTypes);
                 }
             }
-            // for Python func call syntax without the explicit 'call' keyword, check if it is defined in Query scope
+            // for Python function/Groovy closure call syntax without the explicit 'call' keyword, check if it is defined in Query scope
             if (acceptableMethods.size() == 0) {
-                if ( variables.get(methodName) == PythonScopeJpyImpl.CallableWrapper.class) {
-                    for (Method method : PythonScopeJpyImpl.CallableWrapper.class.getDeclaredMethods()) {
+                Class methodCls = variables.get(methodName);
+                if (methodCls == PythonScopeJpyImpl.CallableWrapper.class || methodCls == groovy.lang.Closure.class) {
```

Comment:
Here and below, can we move this test into a very simple helper method?

So here we can say

```
if (isImplicitCallScope(methodCls)) {
...
```

and then (somewhere)

```
private static boolean isImplicitCallScope(final Class cls) {
  return cls == PythonScopeJpyImpl.CallableWrapper.class || cls == groovy.lang.Closure.class;
}
```

And you can choose whatever name you like for `isImplicitCallScope`

Reply:
excellent idea, easy refactoring, i think isImplicitCall is a more fitting name